datafusion/core/tests/parquet/arrow_statistics.rs

```diff
@@ -1724,7 +1808,7 @@ async fn test_boolean() {
     .build()
     .await;
 
-    Test {
+    TestBoth {
```

Comment:
By changing a test to `TestBoth` it will test both row group indexes and data page indexes